locales/wxMaxima/ru.po

```diff
@@ -8643,7 +8637,7 @@ msgid "We can introduce (\"substitute\") them into other equations:\n"
 msgstr "Можно вставить («подставить») их в другие уравнения:\n"
 
 #: ../../examples/examples_gettext.h:1788
-#, fuzzy, c-format
+#, c-format
```

Comment:
This will not work. You should replace it with "no-c-format".

Please try to run "msgfmt -vc -o /dev/null ru.po" to see the error messages.

Reply:
The c-format is autogenerated by msgmerge. So I am unsure if it might automatically reappear at some time.

Reply:
> The c-format is autogenerated by msgmerge. So I am unsure if it might automatically reappear at some time.

Sure. But without removing c-format from two broken strings you cannot compile wxMaxima.

Reply:
What if we merge this PR, then I will fix it this time, but the next time the file will be checked by the committer before PR creation?

Reply:
With merging and fixing I am fine. But instead of adding another task for the translator I would like to resolve the underlying problem.

My guess is that the problem arises only within strings that contain a "%" => Will try to find a way to hinder gettext from using c-format in this file.

Reply:
Thanks a lot! Am currently trying to rename the `.h` file that caused these `c-format` markers to `.list` hoping that this will trick cmake into believing we don't use c code here.